defaults/main.yml

```
@@ -1 +1,13 @@
 clamav_cron_frequency: weekly
+clamav_scan_copy: false
+clamav_scan_exclude_directories:
+  - ^/dev/
+  - ^/proc/
+  - ^/sys/
+  - ^/var/spool/clamav
+clamav_scan_extra_flags: []
+clamav_scan_move: false
+clamav_scan_quarantine_dir: '/var/spool/clamav'
+clamav_scan_quarantine_group: root
+clamav_scan_quarantine_mode: '0750'
```

The single quotes are not necessary here and can be removed.
```suggestion
clamav_scan_quarantine_mode: 0750
```